resources/views/dashboard.twig

```diff
@@ -26,8 +26,11 @@ function deleteTalk(tid) {
     <div class="text-center">
         <i class="fa fa-user-circle text-soft fa-5x"></i>
         <h1 class="font-serif font-thin text-5xl">Welcome to {{ site.title }}!</h1>
-        <p class="text-soft mb-8">It looks like this is your first time here ...</p>
+        <p class="text-soft mb-8">It looks like this is your first time here...</p>
         <a href="{{ url('user_edit', { id: user.id }) }}" class="btn btn-brand">Fill out your profile</a>
+        <br><br>
+        <p class="text-soft mb-8">Click below to delete your account without going any further</p>
+        <a href="{{ url('user_delete') }}" class="btn btn-brand">Delete your account</a>
```

Comment:
Minor nitpick, but for consistency, can we use the same order of attributes as on the link below?

Feel free to use this patch here:

```diff
diff --git a/resources/views/dashboard.twig b/resources/views/dashboard.twig
index cbc69eb5..9221a773 100644
--- a/resources/views/dashboard.twig
+++ b/resources/views/dashboard.twig
@@ -30,7 +30,7 @@ function deleteTalk(tid) {
         <a href="{{ url('user_edit', { id: user.id }) }}" class="btn btn-brand">Fill out your profile</a>
         <br><br>
         <p class="text-soft mb-8">Click below to delete your account without going any further</p>
-        <a href="{{ url('user_delete') }}" class="btn btn-brand">Delete your account</a>
+        <a class="btn btn-brand" href="{{ url('user_delete') }}">Delete your account</a>
     </div>
     {% else %}
     <div class="bg-brand p-8 text-white flex items-center justify-between">
```

Reply:
On the other hand, we use `btn btn-brand` as classes for the link here, but `btn btn-white-o mr-2` below - to the same effect. Maybe we should use semantic classes for these (no need to change anything now).